campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -385,6 +624,19 @@
           }
         {% endif %}
 
+        {% if background_image_animate and background_image %}
```

Comment:
This code does not work well with special images as all images get replaced by the animated image. I'll make a comment for the snippet editors, if you think that this should be fixed lets do it in another PR